htmlcomplete.vim: Fix crash in CompleteTags() (Fix issue #1302)

glacambre opened the pull request:
A crash would sometimes happen in htmlcomplete.vim:251 when the 'context' variable only contained whitespace (split() would try to split context on whitespace and return an empty array, the first cell of which we tried to access). This commit prevents this crash from happening.

dpelle commented on runtime/autoload/htmlcomplete.vim:
```diff
@@ -245,7 +245,7 @@ function! htmlcomplete#CompleteTags(findstart, base)
 	" It can be also value of attribute.
 	" We have to get first word to offer proper completions
-	if context == ''
+	if context == '' || context =~ '\v^\s+$'
```
We could test a single condition:
```
  if context =~ '\v^\s*$'
```

glacambre replied:
You're absolutely right, I don't know why I didn't do that in the first place. I changed it.

Commit:
htmlcomplete.vim: Fix crash in CompleteTags()

A crash would sometimes happen in htmlcomplete.vim:251 when the
'context' variable only contained whitespace (split() would try to split
context on whitespace and return an empty array, the first cell of which
we tried to access). This commit prevents this crash from happening.

brammool commented:
I'll include it now.
